client/src/components/basic/task-instructions/components/Rank/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
file or folder name should follow [JS quality criteria](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md#naming)